### DISCUSSION TITLE
Fix a "Fatal error: Uncaught Error: Class 'phpGPX\phpGPX' not found" error

### DISCUSSION THREAD

**Discussion by richardpbull** [Sibyx/phpGPX#57] [Q&A] [answered]
I know it is not the author's job to help with errors that are not bugs, and due to the user's incompetence, but I shall post this here and if I find a solution, I will post it below in order to help others. 
I have this working well on a localhost MAMP instance. 
However, when copying all files en mass from the localhost example to a shared server, i get: 

Fatal error: Uncaught Error: Class 'phpGPX\phpGPX' not found in /home/customer/www/.../public_html/.../.../admin_gpx_tracks_edit.php:64 Stack trace: #0 {main} thrown in  on ... line 64

require_once('../../vendor/autoload.php');
use phpGPX\phpGPX;

and line 64 is 

$gpx = new phpGPX();

composer.json contains
 "sibyx/phpgpx": "^1.1"

Could any more experience person possibly give some ideas how to resolve this please? Tips very welcome. 
Thank you!

**Comment by Sibyx** [comment, marked as answer]
Hello @richardpbull,
how does your directory structure looks like on shared server? Have you uploaded also the vendor directory (with installed phpGPX)?

**Reply by richardpbull** [reply]
Thank you for your response... 
I didn't know that I had to upload an updated 
`/vendor/composer/autoload_classmap.php`
Uploading that worked. 
Thank you again.
